docs/pages/world/account-delegation.mdx

```diff
@@ -21,9 +23,68 @@ This feature enables multiple use cases.
 
 The most common type of delegation is when a delegator address allows a specific delegatee address to act on its behalf.
 
+### Deploy delegation
+
+The owner of the root namespace (and therefore the `World`) needs to deploy the [`StandardDelegationModule`](https://github.com/latticexyz/mud/blob/main/packages/world-modules/src/modules/std-delegations/StandardDelegationsModule.sol).
```

Comment:
The `StandardDelegationsModule` is used to register a couple pre-build types of limited delegations, namely the `TimeboundDelegation` and `SystemboundDelegation`. You don't need the `StandardDelegationsModule` for unlimited delegations or if you have a custom delegation type.

Reply:
I think we should have examples for how to register an unlimited delegation (the simplest type, doesn't require any additional code to be deployed) and the extend to using custom delegations, using the `StandardDelegationsModule` as an example.

Reply:
I already explain how to do unlimited delegation:

> `delegationControlId`, this is usually the `ResourceId` for a `System` that decides whether an attempt to do something by the delegatee on behalf of the delegator is authorized or not.
  Alternatively, this value can be [`UNLIMITED_DELEGATION`](https://github.com/latticexyz/mud/blob/main/packages/world/src/constants.sol#L23-L25), in which case the delegatee has unlimited authority.
  
But I don't want to do more than that, because it violates least privilege. It's not that installing limited delegation is difficult.